brainglobe_atlasapi/list_atlases.py

```diff
@@ -59,18 +59,22 @@ def get_local_atlas_version(atlas_name):
 def get_all_atlases_lastversions():
     """Read from URL or local cache all available last versions"""
     cache_path = config.get_brainglobe_dir() / "last_versions.conf"
+    custom_path = config.get_brainglobe_dir() / "custom.conf"
 
     if utils.check_internet_connection(
         raise_error=False
     ) and utils.check_gin_status(raise_error=False):
-        available_atlases = utils.conf_from_url(
+        official_atlases = utils.conf_from_url(
             descriptors.remote_url_base.format("last_versions.conf")
         )
     else:
         print("Cannot fetch latest atlas versions from the server.")
-        available_atlases = utils.conf_from_file(cache_path)
-
-    return dict(available_atlases["atlases"])
+        official_atlases = utils.conf_from_file(cache_path)
+    try:
+        custom_atlases = utils.conf_from_file(custom_path)
+    except FileNotFoundError:
+        return dict(official_atlases["atlases"])
+    return dict(official_atlases["atlases"]) | dict(custom_atlases["atlases"])
```

Comment:
```suggestion
    return {**official_atlases["atlases"], **custom_atlases["atlases"]}
```

This might be easier to parse in the future, at first glance, I thought this was doing the equivalent of:
```
return dict(official_atlases["atlases"] if dict(official_atlases["atlases"]) else dict(custom_atlases["atlases"])
```

But that might just be my inexperience with using `|` operator to merge two dicts.